### DISCUSSION TITLE
讨论 | osu! FumoFumo Cup

### DISCUSSION THREAD

**Discussion by giscus** [offfc/offfc.github.io#54] [General]
# 讨论 | osu! FumoFumo Cup

欢迎在这里讨论 OFFC 第二季的内容。管理员请您在下方评论区参与讨论。

https://offfc.github.io/docs/forum/

**Comment by NaughtyChas** [comment]
@kitakore @"My Angel Kuroe"

**Comment by CloneWith** [comment]
大家好啊，我是说的miyu

**Comment by CloneWith** [comment]


**Reply by CloneWith** [reply]
？

**Reply by CloneWith** [reply]
我阐释你的梦

**Comment by CloneWith** [comment]
啊米浴说的道理


         原始邮件
         
       
发件人：                  NaughtyChas         ***@***.***                 
发件时间：         2025年1月8日 16:10 
收件人：                  offfc/offfc.github.io         ***@***.***                 
抄送：                  复予         ***@***.***         ,                Comment         ***@***.***                 
主题：         Re: [offfc/offfc.github.io] 讨论 | osu! FumoFumo Cup (Discussion #54) 




Testing
